addons/sale_stock/tests/test_create_perf.py

```diff
@@ -104,7 +104,7 @@ def test_complex_sales_orders_batch_creation_perf(self):
         # NOTE: sometimes more queries on runbot,
         # do not change without verifying in multi-builds
         # (Seems to be a time-based problem, everytime happening around 10PM)
-        self._test_complex_sales_orders_batch_creation_perf(1502)
+        self._test_complex_sales_orders_batch_creation_perf(2326)
```

Comment:
This seems like a lot. Can you explain it?
The numbers I see here are much more realists https://runbot.odoo.com/runbot/batch/693616/build/12214970

Reply:
I don't think it has to do with the code added in this PR.
I ran this test without it and this is the result :
```python
AssertionError: Query count more than expected for user admin: 2273 > 1502 in _test_complex_sales_orders_batch_creation_perf at sale_stock/tests/test_create_perf.py:138
```
And since the `self.ENTITIES` = 50, it kin of makes sense :man_shrugging:

Reply:
But honestly, even when testing locally, I had inconsistent results, varying slightly (1 or 2 counts) from one run to the other, with no change at all whatsoever.
These tests seem a bit error prone / unreliable, but that's just my 2 cents :wink: 
Plus it will probably require constant update to reflect changes made in many other places in the code base.

Reply:
This test is ran `at_install` (the tag is set by default and not removed). So in order to test it, you should test with only `sale_stock` and its dependencies installed and nothing else (not even the auto installed modules).

Reply:
Indeed, I get the correct results with `--test-tags /sale_stock:TestPERF`.
But not with `--test-file ./addons/sale_stock/tests/test_create_perf.py` :upside_down_face: 
And I don't know which one I should trust ?

Reply:
The results should be the same AFAIK

Reply:
At first I used the values I got from running the tests with `--test-file`, but it seems they generate much more queries than `--test-tag` :man_shrugging: 
Now, I switched to the values from `--test-tag` and it appears they are the ones generated by runbot as well :wink: